src/components/dashboard/FeedItems/CryptoLiteracyNovemberBanner.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Platform, TouchableOpacity } from 'react-native'
+
+import CryptoLiteracyNovemberWeb from '../../../../src/assets/CryptoLiteracyNovember.web.svg'
+import CryptoLiteracyNovemberNative from '../../../../src/assets/CryptoLiteracyNovember.native.svg'
```

Comment:
what's the difference between the svgs?

Reply:
> @TomasGonzalez also in the design the banner is at the top of the page, not at the top of the list

I agreed with @patpedrosa to put it at the bottom.

Reply:
The aspect ratio won't behave the same on mobile and web, they both have width '100%' but the web version also has a height of 100% and the android has a fixed one.

Reply:
i'll accept it but
usually those should be set either directly in the SVG or in its wrapper... see other places where we use SVGs.

Reply:
For js files we could import just 'someModule' and it will be automatically resolved to .web or .native
Could we do the same way with .svg ?

Reply:
I tried this, but doesn't seems to be possible with the current configuration, I'll Investigate if there is something I could do to make it work.

Reply:
@TomasGonzalez okay, please don't waste Your time for this
Just open a new issue regarding web/native assets inclusion support and put it to the IceBox